test/scala/models/BusinessTest.scala

```diff
@@ -0,0 +1,54 @@
+package scala.models
+
+import models.Business
+import org.scalatest.{ FreeSpec, Matchers }
+import play.api.libs.json.Json
+
+import scala.sample.SampleBusiness
+import support.JsonString
+import support.JsonString._
+
+class BusinessTest extends FreeSpec with Matchers with SampleBusiness {
```

Comment:
The businessReads json parser is not a default implementation.  It goes to additional lengths to ensure that it can handle json null values, and to convert empty values to None.  But it doesn't look as if there are unit tests for this ...